challenges/andela_competitive_game.js

```diff
@@ -0,0 +1,46 @@
+function numPlayers(k, scores) {
+  // k is cut off rank cut
+  //  Handle constraints
+  if (
+    !scores ||
+    !k ||
+    k > scores.length ||
+    scores.length < 1 ||
+    scores.length >= Math.pow(10, 5)
+  ) {
+    return 0;
+  }
+
+  // Sort the scores in desceding order
+  scores.sort((a, b) => b - a);
+  console.log(scores);
+
+  const results = [];
+  // Slice array to get the first k items
+  // Remove the items where the score is 0
+  //   Throw if there is an item with value greater than 100 and less than 1
+
+  for (let i = 0; i < k; i++) {
+    const score = scores[i];
+    //   Validate constraints
+    if (0 > score || score > 100) break;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/denpalrius/js-algorithms-and-data-structures/pullRequest?prid=7202625)
```suggestion
    if (0 > score || score > 100) {break;}
```